web/atlas.json

```diff
@@ -2586,7 +2586,6 @@
 {"id": "twvkxq", "submitted_by": "C0rrelationCausation", "name": "New Mexico Flag", "description": "This is a depiction of the flag of the US state of New Mexico. The red Zia symbol is a symbol of the Zia pueblo. The red and green chile are an homage to the state question and answer. (Red or Green? Christmas.) The chile is native to New Mexico and is commonly referred to as Hatch green chile or New Mexico chile. Both red and green chile are a major and unique part of New Mexican cuisine distinguishing it from other Mexican and southwest US cuisines.", "website": "", "subreddit": "/r/NewMexico", "center": [1445.5, 887.5], "path": [[1449.5, 869.5], [1437.5, 870.5], [1437.5, 877.5], [1438.5, 882.5], [1439.5, 891.5], [1439.5, 898.5], [1440.5, 903.5], [1444.5, 905.5], [1449.5, 905.5], [1453.5, 903.5], [1453.5, 896.5], [1453.5, 888.5], [1452.5, 881.5], [1450.5, 878.5], [1449.5, 870.5], [1450.5, 875.5], [1450.5, 870.5]]},
 {"id": "twvkui", "submitted_by": "WADR42", "name": "Dungeons and Daddies", "description": "A DnD podcast about four dads lost in the forgotten realms on a quest to rescue their sons. Starring Freddie Wong, Matt Arnold, Will Campos, Beth May, and Anthony Burch as your Daddy Master.", "website": "https://www.dungeonsanddaddies.com/", "subreddit": "/r/DungeonsAndDaddies", "center": [502.5, 1546.5], "path": [[490.5, 1542.5], [514.5, 1542.5], [514.5, 1549.5], [490.5, 1549.5]]},
 {"id": "twvks0", "submitted_by": "DMgaming100", "name": "Compliance Logo", "description": "Compliance is a Minecraft texturepack available in 32x and 64x resolutions, aiming to provide higher-resolution textures while staying true to the vanilla Minecraft textures", "website": "https://compliancepack.net/", "subreddit": "/r/compliancepack", "center": [1428.5, 119.5], "path": [[1426.5, 128.5], [1429.5, 128.5], [1429.5, 127.5], [1431.5, 127.5], [1431.5, 126.5], [1433.5, 126.5], [1433.5, 125.5], [1435.5, 125.5], [1435.5, 124.5], [1436.5, 124.5], [1436.5, 113.5], [1435.5, 113.5], [1435.5, 112.5], [1433.5, 112.5], [1433.5, 111.5], [1431.5, 111.5], [1431.5, 110.5], [1429.5, 110.5], [1429.5, 109.5], [1426.5, 109.5], [1426.5, 110.5], [1424.5, 110.5], [1424.5, 111.5], [1422.5, 111.5], [1422.5, 112.5], [1420.5, 112.5], [1420.5, 113.5], [1419.5, 113.5], [1419.5, 124.5], [1420.5, 124.5], [1420.5, 125.5], [1422.5, 125.5], [1422.5, 126.5], [1424.5, 126.5], [1424.5, 127.5], [1426.5, 127.5]]},
-{"id": "twvk4u", "submitted_by": "jpepo", "name": "Flag of Croatia", "description": "The flag of Croatia, a country at the crossroads of Central and Southeast Europe.", "website": "https://en.wikipedia.org/wiki/Flag_of_Croatia", "subreddit": "", "center": [98.5, 732.5], "path": [[72.5, 716.5], [72.5, 748.5], [124.5, 748.5], [124.5, 716.5], [72.5, 716.5]]},

```

Comment:
I don't see any duplicate entry for this.

Reply:
Message link https://discord.com/channels/960791635342524496/960982269579911188/960991116134608916 asked for a removal, but I can put it back in if you’d like.

Reply:
There was probably another duplicate beforehand that was cleaned up, so this entry can stay.

Reply:
Added it back.